### PR TITLE
Documentation on cron renewal

### PR DESCRIPTION
**Pull request by hlieberman**
Here's a first pass, @SwartzCr!

### CODE REVIEW

File: docs/using.rst [hunk 1]
```diff
@@ -410,6 +410,13 @@ Renewing certificates
    days). Make sure you renew the certificates at least once in 3
    months.
 
+.. seealso:: Many of the certbot clients obtained through a
+   distribution come with automatic renewal out of the box,
+   such as Debian and Ubuntu versions installed through `apt`,
+   CentOS/RHEL 7 through EPEL, etc.  See `Automated Renewals`_
+   for more details, or consult your distribution's
+   documentation.
+
```
Comment (SwartzCr) [inline docs/using.rst]: I have a vague preference to move this to the part of the section where we talk about cron renewals.
As such I'd move it to be after the paragraph that ends on line 524

File: docs/using.rst [hunk 2]
```diff
@@ -860,6 +891,12 @@ in the config file.
 Log Rotation
 ============
 
+.. note:: Some distributions, including Debian and Ubuntu, disable
+   certbot's internal log rotation in favor of a more traditional
+   logrotate script.  If you are using a distribution's packages and
+   want to alter the log rotation, check `/etc/logrotate.d/` for a
+   certbot rotation script.
+
```
Comment (SwartzCr) [inline docs/using.rst]: Similarly I'd move this to be after the initial paragraph in this section. I think it makes more sense to explain the default log behavior and then have a note about a special case.

**Comment by SwartzCr** [comment]
I fixed my own nits - @bmw wanna review this?

**Comment by hlieberman** [comment]
👍

**Comment by SwartzCr** [comment]
@jsha would you be willing to review this?

File: docs/using.rst [hunk 3]
```diff
@@ -516,6 +516,13 @@ can run on a regular basis, like every week or every day). In that case,
 you are likely to want to use the ``-q`` or ``--quiet`` quiet flag to
 silence all output except errors.
 
+.. seealso:: Many of the certbot clients obtained through a
+   distribution come with automatic renewal out of the box,
+   such as Debian and Ubuntu versions installed through `apt`,
+   CentOS/RHEL 7 through EPEL, etc.  See `Automated Renewals`_
+   for more details, or consult your distribution's
```
Comment (jsha) [inline docs/using.rst]: Is it accurate to say that most distribution-packaged Certbots have documentation somewhere about their cron renewals? I don't see any in the Ubuntu package. I think if the current state is "mostly undocumented" we should leave out this mention of the distribution's documentation.

File: docs/using.rst [hunk 4]
```diff
@@ -611,6 +618,30 @@ The following commands could be used to specify where these files are located::
...
+------------------
+
+Many Linux distributions provide automated renewal when you use the
+packages installed through their system package manager.  The
+following table is an *incomplete* list of distributions which do so,
+as well as their methods for doing so.
+
+If you are not sure whether or not your system has this already
+automated, refer to your distribution's documentation, or check your
+system's crontab and systemd timers.
```
Comment (jsha) [inline docs/using.rst]: We should provide instructions on how to do this, or link to them.

Comment (SwartzCr) [reply docs/using.rst]: Good point - maybe Harlan can take a stab at that? I can't seem to even find the debian documentation

File: docs/using.rst (hunk 3 above)
Comment (hlieberman) [reply docs/using.rst]: The Ubuntu package is the Debian package, so they ship both cron and systemd.  Apparently CentOS and RHEL both do as well, and the combination of those distributions and their derivatives is the bulk of Linux installations out there.

File: docs/using.rst (hunk 4 above)
Comment (hlieberman) [reply docs/using.rst]: Well... to check systemd timers, you'd do `systemctl ilst-timers`.  For cron, you'd check `/etc/crontab` and `/etc/cron.*`.  And, I guess, `/etc/anacrontab`.  That's the standard locations, at least.  But distributions could customize it; it's true for Debian/Ubuntu/CentOS/RHEL, but, for all *nix?

File: docs/using.rst (hunk 3 above)
Comment (jsha) [reply docs/using.rst]: Right, but do the Debian/Ubuntu packages ship documentation? This PR is saying "check the OS documentation," but I think it doesn't exist.

File: docs/using.rst (hunk 4 above)
Comment (jsha) [reply docs/using.rst]: I think providing the typical examples here is good enough.

**Comment by hlieberman** [comment]
Bumping this @SwartzCr.  Can we get this merged?

File: docs/using.rst (hunk 3 above)
Comment (jsha) [reply docs/using.rst]: I think this is still unresolved. Could you just remove the part that says "or consult your distribution's documentation"?

**Comment by jsha** [comment]
Thanks @hlieberman !

**Pull request merged**